app/views/test/ipp/index.html.erb

```diff
@@ -0,0 +1,67 @@
+<% self.title = 'In-person proofing enrollments' %>
+
+<%= content_for(:meta_refresh) { '15' } %>
+
+<h1>In-person proofing enrollments</h1>
+
+<div class="grid-row grid-gap margin-top-2 flex-align-stretch">
+  <div class="tablet:grid-col-12">
+    <% if @enrollments.count == 0 %>
+      <p>
+          There are no recent in-person enrollments.
+      </p>
+    <% else %>
+      <p>
+        Listed below 
+        <%= @enrollments.count == 1 ? 'is' : 'are' %> 
+        the <%= @enrollments.count %> most recent 
+        in-person enrollment<%= @enrollments.count == 1 ? '' : 's' %>.
+      </p>
+
+      <table class="bg-white" border="0" cellpadding="0" cellspacing="0">
+        <thead>
+          <tr>
+            <th class="border-bottom bg-base-lightest padding-x-2 padding-y-1 text-left text-no-wrap"></th>
```

Comment:
I would add "Created At" as the header here

Reply:
~I would add "Created At" as the header here~

I would add "Time since creation" or maybe even "Created" as the header here